proposals/0045-ethereum-network-exporter-id.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: Bridge ID for Ethereum Networks
+Number: 0
+Status: Draft
+Version: 0
+Authors:
+ - Vincent Geddes
+Created: 2023-11-16
+Impact: Trivial
```

Comment:
```suggestion
Impact: High
```

Maybe even `Breaking` since it's breaking the existing `NetworkId::Ethereum` encoding and requires migrations.